lessons/locales_config/en.yml

```diff
@@ -1,55 +0,0 @@
-# Files in the config/locales directory are used for internationalization
```

Comment:
The English file doesn't need to be deleted, can you revert this change? Thanks!